app/components/toc-section.js

```diff
@@ -16,7 +16,7 @@ export default class TocSection extends Component {
   }
 
   get id() {
-    let dasherizedSince = this.result.since.replace(/\./g,'-');
+    let dasherizedSince = this.result.since.replace(/\./g, '-');
```

Comment:
This component poses a technical risk because it uses classic component. When updating to Glimmer component, we want to account for 2 things:

1\. `this.result` and this.displayMobileToc` need to be written as `this.args.result` and `this.args.displayMobileToc` instead.

2\. Line 34 shows mutating the argument `@displayMobileToc`. Let's use DDAU instead.